shader/antialiasing/cmaa2/compute_dispatch_args.comp

```diff
@@ -0,0 +1,46 @@
+#version 460
+
+#extension GL_ARB_separate_shader_objects       : enable
+#extension GL_GOOGLE_include_directive          : enable
+#extension GL_EXT_control_flow_attributes       : enable
+
+#include "cmaa2_common.glsl"
+
+layout(binding = 8, std430) buffer UboWorkingExecuteIndirectBuffer {
+  uvec3 g_workingExecuteIndirectBuffer;
+  };
+
+layout(push_constant, std140) uniform PushConstant {
+  uint setupProcessCandidatesPassFlag;
+  } push;
+
+layout(local_size_x = 1) in;
+
+void main() {
+  if(push.setupProcessCandidatesPassFlag != 0) {
```

Comment:
In C++ side:
```c++
  uint32_t processCandidatesSetupFlag = 1;

  // initialization that is needed only on the first run. Make it run only in the first run
  if(cmaa2.isFirstFrame) {
    cmd.setUniforms(*cmaa2.indirectArgsSetup, cmaa2.indirectArgsSetupUbo, &processCandidatesSetupFlag, sizeof(uint32_t));
    cmd.dispatch(1);
    cmaa2.isFirstFrame = false;
    }
```
You have extra run of this shader for first frame, writing only `g_workingExecuteIndirectBuffer`. Did you mean `processCandidatesSetupFlag = 0` ?

Reply:
Yes, the 'else' path should be run here, the point is just clearing the buffer, but it also could be done explicitly on initialization